### ISSUE TITLE
Improve dropdown component

### ISSUE THREAD

**Issue by rudolfs** [radicle-dev/radicle-upstream#246]
Now that we're actually using the `AdditionalActionsDropdown` component, I noticed that it could be improved:

1) when there are multiple dropdown components on one page, clicking any `...` menu button should close any other open dropdown modals;
2) there seems to be an issue with stacking: if a dropdown menu button comes after another dropdown it will be higher in the Z stack and will appear to be on top of the second dropdown menu
3) it should be possible to specify the menu button background color for the hover state via props (css vars?), otherwise the button blends into the underlying background if the underlying background color matches the hover state background color; alternatively the hover state could be an alpha, so it works on any underlaying background.

<img width="491" alt="Screenshot 2020-03-23 at 13 24 38" src="https://user-images.githubusercontent.com/158411/77316793-54eadc80-6d0a-11ea-8d5e-745bf9a92d97.png">

<img width="317" alt="Screenshot 2020-03-23 at 13 37 18" src="https://user-images.githubusercontent.com/158411/77317742-0dfde680-6d0c-11ea-8d31-3a7a5b5d2f06.png">

When this is fixed, remove any related `force: true` statements from the specs:
```
cypress/integration/user_registration.spec.js
8:        force: true

cypress/integration/project_creation.spec.js
12:        force: true
22:        force: true
```

**Comment by rudolfs** [comment]
#299 seems to have fixed only the stacking problem for the top-bar, we still need a general solution that works even for components that are siblings.

<img width="416" alt="Screenshot 2020-04-23 at 11 22 55" src="https://user-images.githubusercontent.com/158411/80082765-189edc00-8555-11ea-9e1c-d6c050c29c37.png">

**Comment by rudolfs** [comment]
> 3. It should be possible to specify the menu button background color for the hover state via props (css vars?), otherwise the button blends into the underlying background if the underlying background color matches the hover state background color; alternatively the hover state could be an alpha, so it works on any underlaying background.

Will be addressed by: #292
